tools/clang/test/HLSL/eliminate_dynamic_output.hlsl

```diff
@@ -0,0 +1,23 @@
+// RUN: %dxc -Emain -Tvs_6_0 %s | %opt -S -hlsl-dxil-eliminate-output-dynamic | %FileCheck %s
```

Comment:
Need more tests.

Checks for storing multiple times
Checks for multiple returns